### PR TITLE
Add number range and date range scalar filters

### PR DESCRIPTION
**Pull request by Delerme**

### CODE REVIEW

**Comment by Delerme** [comment]
@benmcginnis When using a thirdparty date picker, you can replace the template, do any initialization that thing needs on mount, and use the `setMin` and `setMax` to integrate with the library (has to be in the yyyy-mm-dd format). Similar idea with number range, if you want to use some crazy knob control or something. Does that interface seem flexible enough?

File: src/ui/templates/controls/range.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<fieldset>
+  <legend>{{title}}</legend>
+
+  {{#if minLabel}}
+    <label for="js-yext-range-min-{{name}}">{{minLabel}}</label>
+  {{/if}}
+  <input id="js-yext-range-min-{{name}}" type="number" class="js-yext-range" placeholder="Min" max="{{max}}" data-key="min" value="{{minValue}}">
```
Comment (b1lly) [inline src/ui/templates/controls/range.hbs]: style nit:

can we break these attributes onto new lines?

**Review by b1lly — APPROVED** [review]
LGTM

File: src/ui/templates/controls/range.hbs (hunk 1 above)
Comment (Delerme) [reply src/ui/templates/controls/range.hbs]: done

**Pull request merged**